demo/capture_config.yaml

```diff
@@ -0,0 +1,170 @@
+lcr-modules:
+
+    _shared:
+        lcr-modules: "../"
+        lcr-scripts: "../../lcr-scripts"
+        root_output_dir: "results/"
+        scratch_directory: "scratch/"
+        unmatched_normal_ids:
+            capture--grch37: "TCRBOA7-N-WEX"
+    
+    battenberg:
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_bam: "data/{sample_id}.bam"
+        
+        reference_path:
+            hg38: " --chr_prefixed_genome "                           # Copied from vcf2maf
+            grch37: "/home/lakshay/lcr-modules/battenberg_files/hg19-grch37"     # Copied from vcf2maf
```

Comment:
Another hardcoded path with your personal account. May break when you leave.

Reply:
Its for testing, currently working on removing the need to enter a path altogether